modules/specifying-openshift-installer-log-levels.adoc

```diff
@@ -17,6 +17,6 @@ By default, the {product-title} installer log level is set to `info`. If more de
 +
 [source,terminal]
 ----
-$ ./openshift-install --dir=<installation_directory> wait-for bootstrap-complete --log-level=debug  <1>
+$ ./openshift-install --dir <installation_directory> wait-for bootstrap-complete --log-level=debug  <1>
```

Comment:
$ ./openshift-install --dir <installation_directory> wait-for bootstrap-complete --log-level debug  <1>